features/org.wso2.carbon.ldap.server.server.feature/resources/conf/scimPerson.ldif

```diff
@@ -132,13 +132,33 @@ attributeTypes: ( 1.3.6.1.4.1.37505.1.45
         EQUALITY caseIgnoreMatch
         SUBSTR caseIgnoreSubstringsMatch
         SYNTAX 1.3.6.1.4.1.1466.115.121.1.15{1024} )
+attributeTypes: ( 1.3.6.1.4.1.37505.1.72
+        NAME 'extendedRef'
+        EQUALITY caseIgnoreMatch
+        SUBSTR caseIgnoreSubstringsMatch
+        SYNTAX 1.3.6.1.4.1.1466.115.121.1.15{1024} )
+attributeTypes: ( 1.3.6.1.4.1.37505.1.72
+        NAME 'extendedDisplayName'
```

Comment:
Don't we need to add these attributes to wso2Person object class

Reply:
Since all these custom attributes were specific to SCIM extension schema attributes, I didn't add them to the super class.